client/less/challenge.less

```diff
@@ -138,7 +138,7 @@
 }
 
 .refresh-icon {
-  color: darkgreen;
+  color: #575757;
```

Comment:
thank you @BerkeleyTrue didn't know if I was allowed to make edits there or not.